src/rules/noDupeKeys.js

```diff
@@ -12,13 +12,68 @@ export default (context) => {
     });
   };
 
+  const analizeElement = (element) => {
+    const {type} = element;
+    let value;
+
+    switch (type) {
+    case 'GenericTypeAnnotation':
+      value = element.id.name;
+      break;
+    case 'ObjectTypeAnnotation':
+      /* eslint-disable no-use-before-define */
```

Comment:
Please change `/* eslint-disable */` comments to `// eslint-disable-next-line no-use-before-define`.